package.json

```diff
@@ -3,7 +3,7 @@
   "version": "2.3.0",
   "description": "LoopBack Connector for IBM Cloudant DB",
   "engines": {
-    "node": ">=6"
+    "node": ">=8.9"
```

Comment:
"node": ">=8"

Reply:
another connector does https://github.com/strongloop/loopback-connector-mongodb/blob/master/package.json#L6 but loopback-next has `8.9` but do you think we should follow other connectors?

Reply:
@jannyHou

Reply:
@nabdelgadir @agnes512 Usually "8" is good, loopback-next restricts it to be above "8.9" because lower 8 versions don't work....But for connectors we don't have that problem.